src/network/socket.c

```diff
@@ -350,8 +350,14 @@ get_pasv_socket(struct socket *ctrl_socket, struct sockaddr_storage *addr)
 
 	/* Set it non-blocking */
 
+#ifndef WIN32
```

Comment:
Why did you move it from set_nonblocking_fd? There was already code for WIN32.

Reply:
I mobed it from nonblocking function because it's routive fir all sockets including files and keyboard. Thinking about it I can put it back if I'd test socket for latger than 1024. I'll do that.